com.civicrm.hrjobroles/CRM/Hrjobroles/Upgrader.php

```diff
@@ -138,7 +139,7 @@ private function retrieveCostCentreOtherId() {
       'name' => 'Other',
     ]);
     
-    return $result['id'];
+    return ($result['id'])? $result['id'] : null;
```

Comment:
This will still throw an error, won't it? Perhaps you meant something like this: `!empty($result['id']) ? $result['id'] : null`

Reply:
What i meant is `isset`